authentication/app.js

```diff
@@ -0,0 +1,259 @@
+// See README.md for details about this app
```

Comment:
Would this help disambiguate which README you're referring to?
```suggestion
// See /authentication/README.md for details about this app
```